Variable length array in causing build failure on Mac OS X

The build fails on Mac OS 10.11.5 with following error (relevant excerpt from the build log):

```
qjackctlInterfaceComboBox.cpp:213:21: error: variable length array of non-POD
      element type 'QString'
                        QString hostNames[iNumHostApi];
```

A quick search on Google of course revealed a useful thread on Stack Overflow: https://stackoverflow.com/questions/20461424/error-variable-length-array-of-non-pod-element-type-string

This is probably due to the fact that LLVM/Clang (default compiler on Mac OS X) is a bit stricter than GCC. I've successfully built qjackctl with the suggestion in the Stack Overlfow answer. I will provide a pull request shortly.

--- a/src/qjackctlInterfaceComboBox.cpp
+++ b/src/qjackctlInterfaceComboBox.cpp
@@ -210,7 +210,7 @@ private:
 
 			// Fill HostApi info...
 			const PaHostApiIndex iNumHostApi = Pa_GetHostApiCount();
-			QString hostNames[iNumHostApi];
+			QString *hostNames = new QString[iNumHostApi];
 			for (PaHostApiIndex i = 0; i < iNumHostApi; ++i)
 				hostNames[i] = QString(Pa_GetHostApiInfo(i)->name);
 
@@ -228,6 +228,7 @@ private:
 				}
 			}
 
+			delete [] hostNames;
 			Pa_Terminate();
 		}
 	}

